guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,14 +4,45 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== Entitlement Mode Removed in Favor of Simple Content Access
+Simple Content Access (SCA) is now the only content access mode supported.
+When upgrading Katello, all existing organizations that use entitlement mode will migrate automatically to SCA.
+This change is not reversible.
+
+Toggling SCA from the Organization Edit page, the Organization Create page, and the Organization List page is no longer available.
+
+=== Removed System Purpose-Related Host Statuses
+The following host statuses are removed:
+* Subscription Status (this column has also been removed from the All Hosts table)
+* System Purpose Status
+* SLA Status
+* Role Status
+* Usage Status
+* Addon Status
+
+=== Debian Content Support Expanded
+Import and export are now supported for repositories containing debian-type content.
+Additionally, the host details page will now display more details for debian packages.
 
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
 
-There are no upgrade warnings with Katello {KatelloVersion}.
+Simple Content Access (SCA) is now the only supported content access mode.
+Existing organizations will be automatically migrated on Katello upgrade (see more info above).
+Please note that the 'Subscription - Entitlements' report template does not work with SCA organizations and will never have any rows.
+It will be removed in a future release or release candidate.
+Instead, use the 'Host - Installed Products' report.
+Additionally, please note that SCA-related API and Hammer endpoints and parameters have not yet been removed so you will experience errors if you try to use them.
+They will be removed in a future release or release candidate.
+
+Certain host statuses have been removed (see more info above).
+Please note that system purpose _attributes_ are not being removed, only system purpose-related _host statuses_.
+System purpose _attributes_ (role, usage, release version, etc.) are still important for repository enablement and reporting to the Subscriptions service of the Red Hat Hybrid Cloud Console.
 
 [id="katello-deprecations"]
 == Deprecations
 
-There are no deprecations with Katello {KatelloVersion}.
+Deprecated 'Subscription - Entitlements' report template as it does not work with SCA organizations.
+
```

Comment:
```suggestion
```

Reply:
I think we need the extra space there to logically separate the subs entitlement report template deprecation from the the next one:

![image](https://github.com/theforeman/foreman-documentation/assets/14796566/7359263e-a393-43da-97b0-3698837d4f75)